app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/actions/AllowedUsers.scala

```diff
@@ -24,7 +24,9 @@ import javax.inject.Provider
 
 @ProvidedBy(classOf[AllowedUsersProvider])
 class AllowedUsers(users: Seq[AllowedUser]) {
-  def isAllowed(email: String): Boolean = users.isEmpty || users.exists(user => user.email == email)
+
+  def isAllowed(email: String): Boolean =
+    users.isEmpty || users.exists(user => user.email.equalsIgnoreCase(email))
 
   def getUserFeatures(email: String): Option[Map[String, Boolean]] =
     users.find(_.email == email).map(_.features)
```

Comment:
does the `getUserFeatures` need to be case-insensitive too?

Reply:
Excellent spot!